demisto_sdk/commands/common/constants.py

```diff
@@ -728,7 +728,8 @@ class FileType(Enum):
 TESTS_DIRECTORIES = [
     'testdata',
     'test_data',
-    'data_test'
+    'data_test',
+    'tests_data'
```

Comment:
is this change intended ?

Reply:
Not to this pr but basically yes.
It to not run validation on this folder in content